invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LocationsDetails/LocationsDetails.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Container } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import { InternalLocations } from './components';
+import { Error, Loader, ResultsTable } from '../../../common/components';
+
+export default class LocationsDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.fetchLocations = this.props.fetchLocations;
+  }
+
+  componentDidMount() {
+    this.fetchLocations();
+  }
+
+  openEditor(url) {
+    window.open(`/editor?url=${url}`, url);
```

Comment:
I have also hardcoded the url there, but there is a config variable for it `RECORDS_EDITOR_URL_PREFIX` maybe we can use it.